packages/plugin-dom/Dom.ts

```diff
@@ -24,32 +25,58 @@ export class Dom<T extends DomConfig = DomConfig> extends JWPlugin<T> {
     };
 
     domMap = new DomMap();
+    editable: HTMLElement;
 
     async start(): Promise<void> {
-        const editable = this.editor._originalEditable;
+        const node = this.configuration.target;
+        if (!node) {
+            this.editable = document.createElement('jw-editable');
+            this.domMap.set(this.editor.vDocument.root, this.editable);
+        } else {
+            this.domMap.set(this.editor.vDocument.root, node);
+            this.editable = node.cloneNode(true) as HTMLElement;
+        }
+        const editable = this.editable;
+
+        [...editable.childNodes].forEach(node => {
+            editable.appendChild(node);
+        });
+
+        editable.setAttribute('contenteditable', 'true');
 
         // Construct DOM map from the parsing in order to parse the selection.
-        this.domMap.set(this.editor.vDocument.root, editable);
         const engine = this.editor.parsers.dom as ParsingEngine<Node>;
         for (const [domNode, nodes] of engine.parsingMap) {
             for (const node of nodes) {
                 this.domMap.set(node, domNode);
             }
         }
 
-        // Parse the selection.
-        const selection = editable.ownerDocument.getSelection();
-        const anchorNode = selection.anchorNode;
-        const focusNode = selection.focusNode;
-        if (
-            (editable === anchorNode || editable.contains(anchorNode)) &&
-            (editable === focusNode || editable.contains(focusNode))
-        ) {
-            this.editor.selection.set(this.parseSelection(selection));
-        } else if (this.configuration.autoFocus) {
+        let rangeIsSetted = false;
+        if (node) {
+            // Parse the selection.
+            const selection = node.ownerDocument.getSelection();
+
```

Comment:
Please remove this blank line, otherwise it seems the `Parse the selection` comment only covers the `getSelection` line, which makes no sense.